jnigen/test/bindings_test.dart

```diff
@@ -95,6 +101,22 @@ void main() async {
     ex.delete();
   });
 
+  test('array of the class', () {
+    final ex1 = Example();
```

Comment:
@HosseinYousefi an unnamed constructor 😮

Reply:
I know! Crazy! We had to create it ourselves 🥲